mcda/utils.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import argparse
+import json
+
+
+
+def read_input_matrix(input_matrix_path: str) -> pd.DataFrame():
```

Comment:
Although, the method name is self explanatory but how about a short comment explaining what is the method about?

Reply:
Because it is self explanatory ;-) and I have learned that the best code is a self-explanatory one :-)